java/code/src/com/suse/manager/webui/services/SaltConstants.java

```diff
@@ -21,13 +21,13 @@
 
     private SaltConstants() { }
 
-    public static final String SUMA_STATE_FILES_ROOT_PATH = "/srv/susemanager/salt";
+    public static final String SUMA_STATE_FILES_ROOT_PATH = "@SHAREDSERVDIR@/susemanager/salt";
```

Comment:
I'm not sure how this will cope with local development and deployment. For that we use the file `java/manager-build.xml`, which doesn't replace any of these tokens.
Besides that, if we replace it locally git will mark files as change and developer may wrongly commit it with the replaced values.
In my opinion we should wrap this in the class `ConfigDefaults`, or in a new similar file, and deal with the properties load and is default, and in this way avoid replaces in build time.
It also makes it more flexible, to allow configuration at runtime if needed (@mcalmer I'm not sure if this is needed or even the impact of it)

Reply:
Sounds good.